### PR TITLE
UI: Nicer buttons

### LINKED ISSUE #2252: [Bug] Invisible watch history when using thin mode
**Is your enhancement request related to a problem? Please describe.**
When using thin mode accessing videos in watch history is not possible.

**Describe the solution you'd like**
Change how history is shown when using thin mode.
Im not 100% familiar with this code but based on my understanding this line causes it to skip rendering the thumbnails and ability to click when using thin mode.
https://github.com/iv-org/invidious/blob/edb37c95e1c9dbf1abf2647fcfcd1453c38124c1/src/invidious/views/history.ecr#L36
I would like to see that it changed to check if it is using thin mode and if it is use the title of the video instead of completly skipping.

**Describe alternatives you've considered**
#2122 
Rewrite of thinmode minus the suggestion to removing it

**Additional context**
Requires an invidious account to save watch history.
Requires at least one video in history.
Requires thin mode on.
Result:
Screenshot_20210712_161913

### LINKED ISSUE #3630: [Bug] Only part of the delete button deletes video from playlist
**Describe the bug**
Only part of the delete button deletes video from playlist.

**Steps to Reproduce**
Have a playlist with at least one item.
Click the delete button in the grey area surrounding the garbage can icon.
I would expect the item to be removed, but instead the video is opened up.
The clickable area to delete an item is thus much smaller than a user would expect.

**Additional context**
Relevant code. I tried a bit to fix it using dev tools but couldn't quite get it to work for some reason. My basic idea was to put the submit onto the paragraph.

`invidious/views/components/item.ecr`
```
<% if plid_form = env.get?("remove_playlist_items") %>
  <form data-onsubmit="return_false" action="/playlist_ajax?action_remove_video=1&set_video_id=<%= item.index %>&playlist_id=<%= plid_form %>&referer=<%= env.get("current_page") %>" method="post">
    <input type="hidden" name="csrf_token" value="<%= HTML.escape(env.get?("csrf_token").try &.as(String) || "") %>">
    <p class="watched">
        <button type="submit" style="all:unset" data-onclick="remove_playlist_item" data-index="<%= item.index %>" data-plid="<%= plid_form %>"><i class="icon ion-md-trash"></i></button>
    </p>
  </form>
<% end %>
```

### PR DESCRIPTION
**Pull request by SamantazFox**
Reworked the previous/next buttons to make them more visible, more accessible, and more pleasing to look at!

You can test it at https://test.invidious.io

Closes #2252
Closes #3630

---

Screenshots:
<details><summary>Channel page(s)</summary>

</details> 

<details><summary>Playlists</summary>

</details>

<details><summary>In playlist edit mode  (Please excuse the content, it's purely random garbage)</summary>

image
image

image
image

image

</details>

<details><summary>Watch history</summary>

image
image

</details>

### CODE REVIEW

**Comment by TheFrenchGhosty** [comment]
Looks really better nice job.

This seems like a lot of unrelated code though, you should maybe split this PR in 2.

**Comment by SamantazFox** [comment]
To fix the buttons on the thumbnail, I had to change almost everything in item.cr
So I took the opportunity to also fix thin mode ^^

File: src/invidious/helpers/i18n.cr [hunk 1]
```diff
@@ -165,3 +165,16 @@ def translate_bool(locale : String?, translation : Bool)
     return translate(locale, "No")
   end
 end
+
+def locale_is_rtl?(locale : String?)
+  # Fallback to en-US
+  return false if locale.nil?
+
+  # See https://en.wikipedia.org/wiki/Right-to-left_script#List_of_RTL_scripts
+  return true if locale == "ar" # Arabic
```
Comment (ChunkyProgrammer) [inline src/invidious/helpers/i18n.cr:174]: Might be better to have a set of the rtl languages and using [includes](https://crystal-lang.org/api/1.8.0/Set.html#includes%3F%28object%29%3ABool-instance-method)?

Comment (SamantazFox) [reply src/invidious/helpers/i18n.cr:174]: Ah, right!

**Comment by Fijxu** [comment]
lgtm

**Comment by ChunkyProgrammer** [comment]
Tested, noticed next page doesn't show on search:

Screenshots_2023-04-23-20-18-47

Screenshots_2023-04-23-20-19-12

**Comment by SamantazFox** [comment]
@ChunkyProgrammer regarding the search button, it's a bigger problem than I expected. I'll make a separate Pr to fix that. In essence, it's not caused by the current PR.

**Comment by SamantazFox** [comment]
And some responsive mode screenshots:

<details><summary>expand</summary>

image
image
image

</details>

**Comment by SamantazFox** [comment]
crap...
image

**Comment by SamantazFox** [comment]
Next page on a channel doesn't work x)

**Comment by SamantazFox** [comment]
Re-deployed on https://test.invidious.io

**Comment by SamantazFox** [comment]
Still in testing at https://test.invidious.io

**Comment by SamantazFox** [comment]
> Tested, noticed next page doesn't show on search:

@ChunkyProgrammer So you were right, this PR *did* break search continuation x)
But I'll still make another PR to improve paging anyway!

**Comment by ChunkyProgrammer** [comment]
> > Tested, noticed next page doesn't show on search:
> 
> @ChunkyProgrammer So you were right, this PR _did_ break search continuation x) But I'll still make another PR to improve paging anyway!

I think you'd just need to return items and if the continuation (_) isnt nil for here: https://github.com/iv-org/invidious/blob/507bed6313b49564e53b69a5c9b4d072d1e05e4b/src/invidious/search/processors.cr#L13 
and here: https://github.com/iv-org/invidious/blob/507bed6313b49564e53b69a5c9b4d072d1e05e4b/src/invidious/search/processors.cr#L35

**Comment by SamantazFox** [comment]
@ChunkyProgrammer Yes, that's about it ^^

**Comment by SamantazFox** [comment]
Si I also fixed thin mode in general:
image

**Comment by SamantazFox** [comment]
PR deployed on the test instance!

**Comment by SamantazFox** [comment]
Can't manage to find more bugs :D

**Pull request merged**